README.md

```diff
@@ -30,7 +30,7 @@ Discord Oxygen is a universal client modification with many features and include
       <li>Animated Profile Banner</li>
       <li>Server Boosts (see Oxygen Boosts)</li>
       <li>Unlimited Upload Filesize: Normal 10MB, Discord Nitro 100MB, Discord Oxygen 100,000MB</li>
-      <li>Unlimited Servers: normal 100 Servers, Discord Nitro 200 Servers, Discord Oxygen ∞ Servers<br>(after more than 10,000 Servers your PC might get a bit laggy)</li>
+      <li>Unlimited Servers: normal 100 Servers, Discord Nitro 200 Servers, Discord Oxygen ∞ Servers<br>(after 10,000 servers your PC might lag)</li>
```

Comment:
Should be left as “more than 10000”

Reply:
"after more than 10000" sounds gramatically incorrect
"after 10000" sounds better